filesystem/local.go

```diff
@@ -143,6 +143,15 @@ func (r *Local) Get(file string) (string, error) {
 	return string(data), nil
 }
 
+func (r *Local) GetBytes(file string) ([]byte, error) {
```

Comment:
S: The `Get` method can call this method.

Reply:
> S: The `Get` method can call this method.

I will optimize this.